### PR TITLE
Fixes some typos and grammar

### PR DESCRIPTION
**Pull request by IrkallaEpsilon**
Love the new shard plants. Description had a few typos and grammar errors. Should fix these.

## About The Pull Request
<details>
<summary>
	Fixes amerdian shard plant description. It had some typos and odd grammar in it.
</summary>
<hr>

## Changelog
:cl:
fix: Grammar on ameridian shard plant fixed as well as typos.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/flora/flora_potted.dm [hunk 1]
```diff
@@ -318,14 +318,14 @@
 		shield_level += 1
 	if(shield_level == 3)
 		icon_state = "ameridian_pot_shield_powered"
-		desc = "A contained shiney shard of ameridian, uses a posi sound for its power and has some osium for emp shielding making it super safe.."
+		desc = "A contained shiny ameridian shard. Uses a positronic cell for power supply and an osmium coating for emp shielding making it super safe."
```
Comment (r4d6) [inline code/game/objects/structures/flora/flora_potted.dm:321]:
```suggestion
		desc = initial(desc)
```

File: code/game/objects/structures/flora/flora_potted.dm [hunk 2]
```diff
@@ -318,14 +318,14 @@
...
 		return
 	if(shield_level == 2)
 		icon_state = "ameridian_pot_shield_unpowered"
-		desc = "A contained shiney shard of ameridian, uses a posi sound for its power and has some osium for emp shielding making it super safe.."
+		desc = "A contained shiny ameridian shard. Uses a positronic cell for power supply and an osmium coating for emp shielding making it super safe."
```
Comment (r4d6) [inline code/game/objects/structures/flora/flora_potted.dm:325]:
```suggestion
		desc = "A contained shiny ameridian shard. Uses a positronic cell for power supply and an osmium coating for emp shielding making it super safe."
```
```suggestion
		desc = initial(desc)
```

File: code/game/objects/structures/flora/flora_potted.dm [hunk 3]
```diff
@@ -318,14 +318,14 @@
...
 		return
 	if(shield_level == 1)
 		icon_state = "ameridian_pot_shieldless"
-		desc = "A contained shiney shard of ameridian, uses a posi sound for its power and has some osium for emp shielding making it super safe. Its shield light level is set to its lowest level making it still contained but look nice."
+		desc = "A contained shiny ameridian shard. Uses a positronic cell for power supply and an osmium coating for emp shielding making it super safe. Its shield light level is set to its lowest level making it still contained but look nice."
```
Comment (r4d6) [inline code/game/objects/structures/flora/flora_potted.dm:329]:
```suggestion
		desc = initial(desc)
		desc += "Its shield light level is set to its lowest level making it still contained but look nice."
```

**Comment by r4d6** [comment]
So.... sorry, but I ended up changing the description while fixing the change icon thing...

Sorry.... https://github.com/sojourn-13/sojourn-station/pull/3287

**Comment by IrkallaEpsilon** [comment]
> So.... sorry, but I ended up changing the description while fixing the change icon thing...
> 
> Sorry.... #3287

Works for me

**Comment by cdb-is-not-good** [comment]
Should be good after the conflicts are resolved

**Pull request closed without merging**